source/thread_br_conn_handler.c

```diff
@@ -98,12 +98,15 @@ static void thread_br_conn_handler_border_router_startup_attempt(void)
 void thread_br_conn_handler_thread_connection_update(bool status)
 {
     thread_br_handler.thread_connection_ready = status;
+    tr_debug("mesh0 connected: %d", status);
```

Comment:
Isn't this debug message a little bit confusing? If I understand right, depending on status, mesh is either connected or disconnected.